### PR TITLE
Created doc: RETURN

### PR DESCRIPTION
**Pull request by randomJoe211**

### CODE REVIEW

File: docs-2.0/3.ngql-guide/8.clauses-and-options/return.md [hunk 1]
```diff
@@ -0,0 +1,233 @@
+# RETURN
+
+The `RETURN` clause defines the output of openCypher statements such as `MATCH` or `UNWIND`. To return multiple fields, separate them with commas.
+
+## OpenCypher compatibility
+
+This topic applies to the openCypher syntax in nGQL only. For nGQL extensions, use [`YIELD`](yield.md).
```
Comment (whitewum) [inline docs-2.0/3.ngql-guide/8.clauses-and-options/return.md:7]: This topic applies to the openCypher syntax in Nebula Graph. WDYT

**Review by whitewum — COMMENTED** [review]
Wait for pr https://github.com/vesoft-inc/nebula-graph/pull/760/files
and you can update

File: docs-2.0/3.ngql-guide/8.clauses-and-options/return.md (hunk 1 above)
Comment (randomJoe211) [reply docs-2.0/3.ngql-guide/8.clauses-and-options/return.md:7]: At first sight, I thought it is a good idea. But on second thought, this is the nGQL guide and the openCypher part still belongs to nGQL 2.0... So why not nGQL

nGQL 2.0 = nGQL extensions + openCypher, am I right?

**Comment by whitewum** [comment]
You can update this pr now

**Comment by randomJoe211** [comment]
> You can update this pr now

@whitewum Done.

**Review by cooper-lzy — APPROVED** [review]

File: docs-2.0/3.ngql-guide/8.clauses-and-options/return.md [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+`RETURN` defines the output of an nGQL query. To return multiple fields, separate them with commas.
+
+`RETURN` can lead a clause or a statement:
+
+* A `RETURN` clause works in openCypher statements in nGQL, such as `MATCH` or `UNWIND`.
+* A `RETURN` statement works independently to output the result of an expression.
+
+## OpenCypher compatibility
+
+This topic applies to the openCypher syntax in nGQL only. For nGQL extensions, use [`YIELD`](yield.md).
```
Comment (whitewum) [inline docs-2.0/3.ngql-guide/8.clauses-and-options/return.md:12]: `This topic applies to the openCypher syntax in nGQL only. For nGQL extensions, use YIELD.`
You can delete this now.

Comment (randomJoe211) [reply docs-2.0/3.ngql-guide/8.clauses-and-options/return.md:12]: Not yet:

image

**Review by whitewum — APPROVED** [review]

**Pull request merged**